CMakeLists.txt

```diff
@@ -43,7 +45,11 @@ if (ENABLE_RHSM_SUPPORT)
     add_definitions (-DRHSM_SUPPORT)
 endif ()
 pkg_check_modules (CHECK REQUIRED check)
-pkg_check_modules (REPO REQUIRED librepo)
+
+if (NOT APPLE)
```

Comment:
Not sure if we want that

Reply:
It does affect anything at all, it only makes my work easier.